include/api/CForecastRunner.h

```diff
@@ -68,10 +69,22 @@ class API_EXPORT CForecastRunner final : private core::CNonCopyable {
     static const size_t DEFAULT_EXPIRY_TIME = 14 * core::constants::DAY;
 
     //! max memory allowed to use for forecast models
-    static const size_t MAX_FORECAST_MODEL_MEMORY = 20971520; // 20MB
+    static const size_t MAX_FORECAST_MODEL_MEMORY = 20971520ull; // 20MB
+
+    // Note: This value measures the size in memory, not the size of the persistence,
+    // which is likely higher and would be hard to calculate upfront
+    //! max memory allowed to use for forecast models persisting to disk
```

Comment:
Only the last line of this begins with `//!`.  All the lines should be consistent.